.gitignore

```diff
@@ -1,2 +1,4 @@
 bench/bench
 build/
+
+*.sw?
```

Comment:
Ah, whoops. This wasn't meant to be committed, but hey, my fellow Vim users probably won't mind :). I can remove it if you want.